src/frontend/src/lib/constants/analytics.contants.ts

```diff
@@ -21,3 +21,8 @@ export const TRACK_COUNT_CONVERT_CKERC20_TO_ERC20_SUCCESS = 'ic_ckerc20_to_erc20
 export const TRACK_COUNT_CONVERT_CKERC20_TO_ERC20_ERROR = 'ic_ckerc20_to_erc20_error_count';
 export const TRACK_COUNT_IC_SEND_SUCCESS = 'ic_send_success_count';
 export const TRACK_COUNT_IC_SEND_ERROR = 'ic_send_error_count';
+
+// WalletConnect
+export const TRACK_COUNT_WALLET_CONNECT_MENU_OPEN = 'wallet_connect_menu_open_count';
+export const TRACK_COUNT_WALLET_CONNECT_QR_CODE = 'wallet_connect_qr_code_count';
+export const TRACK_COUNT_WALLET_CONNECT = 'wallet_connect_connect_count';
```

Comment:
maybe a similar name to `WalletConnectPaired`? eheh

Reply:
Not sure if you suggest to rename those envets or add another one when the WalletConnect is paired?

Those metrics a solely meant to track how the process is initiated in order to discover if:

- Do people use the QRCode reader to connect WalletConnect?
- Or do they copy/paste the URI
- Do they access it through the menu but, do not start a connection?
- Or do they use the routing - `/wc/` tracked by pages statistics - to access Oisy from WC / from elsewhere?

Reply:
what i meant was that `wallet_connect_connect_count` could be called just `wallet_connect_count`, no?

Reply:
Ah absolutely! I was confused because you wrote `WalletConnectPaired` - i.e. "paired".

Let me fix this typo.

Reply:
Done